travis/rebuild_bundles.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+tooling_jar=./travis/tooling/tooling-1.0.4-SNAPSHOT-jar-with-dependencies.jar
+rebuild_log=./travis/tooling/rebuildRun.log
+
+# Remove EXM165 because it is unfinished
+rm ./fhir4/input/resources/measure/measure-EXM165_FHIR4-8.5.000.json
```

Comment:
NIT: Maybe we can remove the corresponding `cql` file for this as well? If we don't we get a log message in the console: 
![image](https://user-images.githubusercontent.com/16297930/78150900-ebec1e80-7405-11ea-9099-59bc24a65db9.png)

Not a huge deal though.